modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/service/test/ObjectActionLocalServiceTest.java

```diff
@@ -1242,6 +1255,91 @@ public void testAddObjectActionWithMoreThanOneObjectEntry()
 	@Test
 	public void testAddObjectActionWithSystemObject() throws Exception {
 
+		// Account entry system object
+
+		ObjectDefinition accountEntryObjectDefinition =
+			_objectDefinitionLocalService.fetchObjectDefinition(
+				TestPropsValues.getCompanyId(),
+				AccountEntry.class.getSimpleName());
+
+		// Add object action to send an email notification after updating
+		// account entry
+
+		ObjectAction objectAction1 = _addNotificationObjectAction(
+			ObjectActionTriggerConstants.KEY_ON_AFTER_UPDATE,
+			accountEntryObjectDefinition,
+			"[%ACCOUNTENTRY_AUTHOR_EMAIL_ADDRESS%]");
+
+		List<NotificationQueueEntry> notificationQueueEntries;
+
+		String originalName = PrincipalThreadLocal.getName();
+		PermissionChecker originalPermissionChecker =
+			PermissionThreadLocal.getPermissionChecker();
+
+		User user = TestPropsValues.getUser();
+
+		try {
+			PrincipalThreadLocal.setName(user.getUserId());
+			PermissionThreadLocal.setPermissionChecker(
+				PermissionCheckerFactoryUtil.create(user));
```

Comment:
Hey @gabrielwas I believe we don't need this because the user created at: 

`User omniadminUser = UserTestUtil.addOmniadminUser();` 

is already a different one that is logged in, please remove this.

Thank you!